hedera-node/src/main/java/com/hedera/services/ledger/accounts/HederaAccountCustomizer.java

```diff
@@ -66,8 +75,16 @@ public void customizeSynthetic(final ContractCreateTransactionBody.Builder op) {
 			op.setAutoRenewPeriod(Duration.newBuilder()
 					.setSeconds((long) changes.get(AccountProperty.AUTO_RENEW_PERIOD)));
 		}
-		if (changes.containsKey(AccountProperty.PROXY)) {
-			op.setProxyAccountID(((EntityId) changes.get(AccountProperty.PROXY)).toGrpcAccountId());
+		if (changes.containsKey(AccountProperty.STAKED_ID)) {
+			final long id = (long) changes.get(AccountProperty.STAKED_ID);
+			if (id <= 0) {
```

Comment:
Should this be strictly `< 0`? (If I understood, `stakedId == 0` meant this account does not stake to anything.)

Reply:
Ah yes .. 😞  Changed it . Thanks @tinker-michaelj